pkg/sentry/inet/inet.go

```diff
@@ -153,3 +153,21 @@ type Route struct {
 	// GatewayAddr is the route gateway address (RTA_GATEWAY).
 	GatewayAddr []byte
 }
+
+// StatSNMPIP describes Ip line of /proc/net/snmp.
```

Comment:
Please add a reference to the corresponding linux header where these are defined.

Reply:
OK, will add.